conans/client/migrations_settings.py

```diff
@@ -1304,4 +1304,100 @@
 settings_1_27_0 = settings_1_26_1
 settings_1_27_1 = settings_1_27_0
 
-settings_1_28_0 = settings_1_27_1
```

Comment:
@memsharded should this be updated by the current PR?

Reply:
Yes, it is good, it is forced to fail if not updated, because otherwise we were failing to do so, and will only detect later in CI